fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1759,6 +1759,15 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, varType = VariableAnnotation.EXPERIMENTAL)
     public static boolean enable_cpu_hard_limit = false;
 
+    @ConfField(mutable = true, description = {
+            "当BE内存用量大于该值时，查询会进入排队逻辑，默认值为-1，代表该值不生效。取值范围0~1的小数",
+            "When be memory usage bigger than this value, query could queue, "
+                    + "default value is -1, means this value not work. Decimal value range from 0 to 1"})
+    public static double query_queue_by_be_used_memory = -1;
```

Comment:
```suggestion
    public static double query_queue_by_be_used_memory = -1;
```
Better add unit in config name, like `xxx_mb` or `xxx_bytes`

Reply:
this is a 0~1 float value, so it's not a mb or bytes